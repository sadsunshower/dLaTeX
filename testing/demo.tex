\documentclass{article}

\begin{document}

Strictly for flexing purposes:

\[
    f(x) = sin(x+y)
\]

\[
    g(x,y) = (\frac{x+y}{x^2+y^2})
\]

Because we have $y=e^-x$, we find that:

Alternatively, we can set \alpha=0

\end{document}