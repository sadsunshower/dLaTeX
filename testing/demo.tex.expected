This is dLaTeX version 0.1
{"messages": [{"severity": "error", "message": "Math mode command / macro used outside of math mode. Did you forget a $ or \\[?", "line": "Alternatively, we can set \\alpha=0", "num": 17, "marker": [31]}, {"severity": "warning", "message": "This looks like it isn't a math operator (\\sin, \\cos, etc.) when it should be", "line": "    f(x) = sin(x+y)", "num": 7, "marker": [11]}, {"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "    g(x,y) = (\\frac{x+y}{x^2+y^2})", "num": 11, "marker": [13, 34]}, {"severity": "warning", "message": "Remember to add curly braces {} around powers with more than one symbol.", "line": "Because we have $y=e^-x$, we find that:", "num": 14, "marker": [20]}], "error": true}
